test/e2e/specs/multi-block-selection.test.js

```diff
@@ -117,4 +117,25 @@ describe( 'Multi-block selection', () => {
 		// Verify selection
 		await expectMultiSelected( blocks, true );
 	} );
+
+	it( 'should speak() number of blocks selected with multi-block selection', async () => {
+		await clickBlockAppender();
+		await page.keyboard.type( 'First Paragraph' );
+		await insertBlock( 'Paragraph' );
+		await page.keyboard.type( 'Second Paragraph' );
+		await insertBlock( 'Paragraph' );
+		await page.keyboard.type( 'Third Paragraph' );
+
+		// Multiselect via keyboard.
+		await pressWithModifier( META_KEY, 'a' );
+		await pressWithModifier( META_KEY, 'a' );
+
+		// TODO: It would be great to do this test by spying on `wp.a11y.speak`,
+		// but it's very difficult to do that because `wp.a11y` has
+		// DOM-dependant side-effect setup code and doesn't seem straightforward
+		// to mock. Instead, we check for the DOM node that `wp.a11y.speak()`
+		// inserts text into.
```

Comment:
I thought that `jest.spyOn` didn't change the spied method's implementation?

Reply:
It doesn't, but the issue is that we can't spy on the method (that I tried) because it's only available inside the page context. I tried:

```js
const a11y = await page.evaluate( () => wp.a11y );
const speakSpy = jest.spyOn( a11y, 'speak; );
```

but it didn't work. 🤷‍♂️